cylc/flow/task_job_mgr.py

```diff
@@ -719,7 +719,6 @@ def _set_retry_timers(itask, rtconfig=None, retry=True):
                 submit_delays = itask.platform['submission retry delays']
             # TODO: same for execution delays?
 
-        if retry:
```

Comment:
Click on the blue button up here :point_left: to see more of the code... there's another `if retry` just above this other `if retry` :+1: